### PR TITLE
[ONNX] Extend ONNX Frontend with Function LessOrEqual

### PR DESCRIPTION
**Pull request by YaritaiKoto**
### Details:
 - Implemented ONNX frontend LessOrEqual operator and corresponding tests.

### Tickets:
 - https://github.com/openvinotoolkit/openvino/issues/20546

### CODE REVIEW

File: src/frontends/onnx/frontend/src/op/less_or_equal.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
```
Comment (gkrivor) [inline src/frontends/onnx/frontend/src/op/less_or_equal.cpp:5]: This declaration should be only in headers, remove please.

File: src/frontends/onnx/frontend/src/op/less_or_equal.cpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+OPENVINO_SUPPRESS_DEPRECATED_START
+
+#include "default_opset.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+OutputVector less_or_equal(const Node& node) {
+    const auto& input = node.get_ng_inputs();
```
Comment (gkrivor) [inline src/frontends/onnx/frontend/src/op/less_or_equal.cpp:17]: Looks like it also has a difference only in supported format. Could you add a check as we did before?
```cpp
    NGRAPH_CHECK(A.get_element_type() != ov::element::bf16 && B.get_element_type() != ov::element::bf16,
                 "The input data bfloat16 isn't supported in opset 12");
```

**Review by gkrivor — CHANGES_REQUESTED** [review]
Hi, good job, thanks! Could you apply some changes, please?

File: src/frontends/onnx/frontend/src/op/less_or_equal.cpp (hunk 1 above)
Comment (YaritaiKoto) [reply src/frontends/onnx/frontend/src/op/less_or_equal.cpp:5]: Oops, forgot to delete that after copy.

**Comment by gkrivor** [comment]
build_jenkins

**Comment by gkrivor** [comment]
build_jenkins

**Review by gkrivor — APPROVED** [review]

**Pull request merged**